volta_base/launch/volta_sensors.launch

```diff
@@ -1,29 +1,37 @@
 <?xml version="1.0"?>
 <launch>
 
-    <arg name="intel_D435_camera" default="false"/>
-    <arg name="rplidar_a3" default="false"/>
-    <arg name="um7_imu" default="false"/>
+    <arg name="orbbec_camera" default="true"/>
+    <arg name="rplidar_a3" default="true"/>
+    <arg name="um7_imu" default="true"/>
 
     <group if="$(arg um7_imu)">
         <node pkg="um7" type="um7_driver" name="imu_driver">
-            <param name="port" value="/dev/IMU"/>
+            <param name="port" value="/dev/imu"/>
             <param name="mag_updates" value="false"/>
             <param name="tf_ned_to_enu" value="false"/>
             <param name="frame_id" value="imu"/>
             <remap from="imu" to="imu"/>
         </node>
     </group>
 
-    <!-- Include Intel D435 Driver -->
-    <group if="$(arg intel_D435_camera)">
-
+    <!-- Include Orbbec Camera Driver -->
+    <group if="$(arg orbbec_camera)">
+      <include file="$(find astra_camera)/launch/astrapro.launch"></include>
```

Comment:
Can add here the comment linking to astra ros package - such that it becomes easy for the user to get the repo